### PR TITLE
Update state pruning command

### PR DESCRIPTION
**Pull request by EmilFattakhov**
Small documentation update to reflect https://github.com/subspace/subspace/pull/2177

Important: will only work on gemini-3g

### CODE REVIEW

File: docs/protocol/substrate-cli.mdx [hunk 1]
```diff
@@ -220,7 +220,7 @@ We will be downloading two files for your respective operating system.
   .\subspace-node-windows-x86_64-skylake-gemini-3f-2023-oct-06.exe `
     --chain gemini-3f `
     --blocks-pruning 256 `
-    --state-pruning archive `
+    --state-pruning archive-canonical-canonical `
```
Comment (jim-counter) [inline docs/protocol/substrate-cli.mdx:223]:
```suggestion
    --state-pruning archive-canonical `
```

File: docs/protocol/substrate-cli.mdx [hunk 2]
```diff
@@ -339,7 +339,7 @@ We will be downloading two files for your respective operating system.
   ./subspace-node-macos-aarch64-gemini-3f-2023-oct-06 \
     --chain gemini-3f \
     --blocks-pruning 256 \
-    --state-pruning archive \
+    --state-pruning archive-canonical-canonical \
```
Comment (jim-counter) [inline docs/protocol/substrate-cli.mdx:342]:
```suggestion
    --state-pruning archive-canonical \
```

**Review by jim-counter — CHANGES_REQUESTED** [review]

File: docs/protocol/substrate-cli.mdx (hunk 1 above)
Comment (EmilFattakhov) [reply docs/protocol/substrate-cli.mdx:223]: Oops, how did that happen!

**Review by jim-counter — APPROVED** [review]
Thanks Emil.

**Pull request merged**